### PR TITLE
Display the application name

### PR DESCRIPTION
**Pull request by beatrice-acasandrei**
Display application for expanded row with one run:
Screenshot 2023-09-07 at 11 12 11

Display application for expanded row with multiple runs:
Screenshot 2023-09-07 at 11 12 48

For testing just use the [fakedata](https://deploy-preview-531--mozilla-perfcompare.netlify.app/#/compare-results/?fakedata=true).

### CODE REVIEW

File: src/components/CompareResults/beta/RevisionRowExpandable.tsx [hunk 1]
```diff
@@ -73,7 +75,11 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
           <Divider />{' '}
         </div>
         {!shouldDisplayGraph && (
-          <div className={`${styles.bottomSpace}`}>{singleRun}</div>
+          <div className={`${styles.bottomSpace}`}>
+            {singleRun} <br />
+            <b>Base application</b>: {baseApplication} <br />
+            <b>New application</b>: {newApplication}{' '}
```
Comment (julienw) [inline src/components/CompareResults/beta/RevisionRowExpandable.tsx:81]: Should we avoid displaying them if it's not present?
image

File: src/components/CompareResults/beta/RevisionRowExpandable.tsx [hunk 2]
```diff
@@ -73,7 +75,11 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
           <Divider />{' '}
         </div>
         {!shouldDisplayGraph && (
-          <div className={`${styles.bottomSpace}`}>{singleRun}</div>
+          <div className={`${styles.bottomSpace}`}>
+            {singleRun} <br />
```
Comment (julienw) [inline src/components/CompareResults/beta/RevisionRowExpandable.tsx:79]: Can we use 2 `div`s instead of  `br`?

**Review by julienw — COMMENTED** [review]
Here are a few comments, thanks!

File: src/components/CompareResults/beta/RunValues.tsx [hunk 3]
```diff
@@ -23,12 +24,26 @@ const styles = {
...
+    measurementUnit,
+  } = props.revisionRuns;
   return (
     <div className={styles.container}>
-      <div>
-        <b>{name}:</b> {median} ms
-      </div>
+      {application ? (
+        <div>
+          <b>{name}:</b> {median} {measurementUnit} ({application})
```
Comment (julienw) [inline src/components/CompareResults/beta/RunValues.tsx:40]: Is there a way to show this with the fake data?

Comment (beatrice-acasandrei) [reply src/components/CompareResults/beta/RunValues.tsx:40]: There is an example with the fakedata, look for B: 4 N:5.

Comment (julienw) [reply src/components/CompareResults/beta/RunValues.tsx:40]: I don't see it for this specific one, am I missing something?

image

However I do see it for others that also have B:4 N:5.

Can you please check that it's normal that `cpstartup content-process-startup 9d5066525489` doesn't show it?

**Review by julienw — APPROVED** [review]
I'm approving because I see the application name for some graphs now, but please make sure that the fact we don't see it for `9d5066525489` is expected!

File: src/components/CompareResults/beta/RunValues.tsx (hunk 3 above)
Comment (beatrice-acasandrei) [reply src/components/CompareResults/beta/RunValues.tsx:40]: For '[cross_origin_pageload](https://firefox-source-docs.mozilla.org/testing/perfdocs/talos.html#cross-origin-pageload) [9d5066525489](https://treeherder.mozilla.org/jobs?repo=mozilla-central&revision=9d50665254899d8431813bdc04178e6006ce6d59)' you can see the application, but for '[cpstartup](https://firefox-source-docs.mozilla.org/testing/perfdocs/talos.html#cpstartup) content-process-startup [9d5066525489](https://treeherder.mozilla.org/jobs?repo=mozilla-central&revision=9d50665254899d8431813bdc04178e6006ce6d59)' there's no application available.

**Comment by beatrice-acasandrei** [comment]
> I'm approving because I see the application name for some graphs now, but please make sure that the fact we don't see it for `9d5066525489` is expected!

Yes, it's expected.

**Review by esanuandra — APPROVED** [review]

**Pull request merged**